_test/shared/matlab_xunit/xunit/TestSuite.m

```diff
@@ -242,7 +242,7 @@ function delete(self)
             %   constructs a TestSuite object containing only the test case
             %   named 'testA' found in the TestCase subclass MyTests.
 
-            if isdir(name)
+            if exist(name, 'dir') == 7
```

Comment:
We don't need the `==7` here. The `if` statement will implicitly cast to a logical